### PR TITLE
chore(searchable-dropdown): web component update

### PR DESCRIPTION
**Pull request by AndrejNikolicEq**
# Component Development
### Type

- [ ] Feature
- [ ] Fix
- [ ] Hotfix (should release ASAP)
- [x] Maintenance (deps only)

### Description of assignment
Maintenace, update of web component and other dependencies, to make disabled and error list item visible again

### CODE REVIEW

File: packages/searchable-dropdown/README.md
Comment (eikeland) [inline packages/searchable-dropdown/README.md:1]: Why remove all content here? its needed for npm.js

**Review by eikeland — CHANGES_REQUESTED** [review]

File: packages/searchable-dropdown/README.md
Comment (AndrejNikolicEq) [reply packages/searchable-dropdown/README.md:1]: Duplicate content, already generated from package
image

Comment (eikeland) [reply packages/searchable-dropdown/README.md:1]: But we need some in the readme file for use on the package npmjs site:
https://www.npmjs.com/package/@equinor/fusion-react-person

Comment (AndrejNikolicEq) [reply packages/searchable-dropdown/README.md:1]: Ok, then will return 👍

Comment (eikeland) [reply packages/searchable-dropdown/README.md:1]: We need what was there, i would rather remove the readme file from the story if its to much there.
on line 17 in `storybook/src/stories_v7/searchable-dropdown/searchable-dropdown.mdx`

**Review by eikeland — CHANGES_REQUESTED** [review]

**Review by eikeland — APPROVED** [review]

**Pull request merged**